OracleTuxedo/OracleTSAM/dockerfiles/12.2.2/Dockerfile

```diff
@@ -100,10 +100,8 @@ RUN mkdir /etc/envs && \
       bash -c "echo . /etc/envs/bashrc >> /etc/bashrc" && \
       bash -c "echo . /etc/envs/bashrc >> /u01/oracle/.bashrc" && \
       cd /u01/oracle && \
-      jar xf /u01/$ORACLT_PKG1 && \
-      jar xf /u01/$ORACLT_PKG2 && \
-      chmod -R +x instantclient_* && \
-      mv /u01/oracle/instantclient* /u01/oracle/instantclient && \
+      rpm -i --nodeps /u01/$ORACLT_PKG1 /u01/$ORACLT_PKG2 && \
```

Comment:
Updated to use `yum localinstall`.
Just wanted to use basiclite version of instant client to make the downloads / image size as small as possible. But neither `rpm -i basiclite.rpm sqlplus.rpm` nor `yum installlocal basiclite.rpm sqlplus.rpm` let me install. It keep complaining that dependency is not satisfied, which looks like a bug. So I had to use `--nodeps` option as I know `libaio` is the only dependency and I had it.